test/nextcloud/1.1.0/templates/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       initContainers:
         - name: init-postgresdb
           image: busybox:latest
-          command: ['sh', '-c', "until nslookup {{ template "nextcloud.fullname" . }}-postgres; do echo waiting for postgres; sleep 2; done"]
+          command: ['sh', '-c', "until nc -w 5 -vz {{ template "nextcloud.fullname" . }}-postgres 5432; do echo waiting for postgres; sleep 2; done"]
```

Comment:
There is a `_ 5432` in this command? Is that supposed to be there? Looks weird.

Reply:
Yes it is, we check for 2 things here:
1) Is postgres service available which would be DNS resolving
2) Is postgres listening on 5432 or not

Reply:
got it, the variable for the container name is stored in between the `{{ }}`. For whatever reason I associated the `5432` to the name of the container.